src/main/webapp/assets/modules/employer-dashboard/popupEmailDirective.js

```diff
@@ -0,0 +1,12 @@
+techlooper.directive("dailySummeryEmail", function () {
+  return {
+    restrict: "E",
+    replace: true,
+    templateUrl: "modules/employer-dashboard/dailySummeryEmail.html",
+    scope: {
+      composeEmail: "="
+    },
+    link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.<br>'scope' is defined but never used.